AdMobAdapter/src/main/java/com/chartboost/mediation/admobadapter/AdMobAdapter.kt

```diff
@@ -381,11 +381,20 @@ class AdMobAdapter : PartnerAdapter {
                 val adview = AdView(context)
                 val partnerAd = PartnerAd(
                     ad = adview,
-                    details = emptyMap(),
+                    details = mutableMapOf(),
                     request = request,
                 )
 
-                adview.setAdSize(getAdMobAdSize(request.size))
+                val adSize = getAdMobAdSize(context, request.size, request.format.key == "adaptive_banner")
+
+                if(request.format.key == "adaptive_banner") {
+                    (partnerAd.details as MutableMap).let {
```

Comment:
I don't like this cast. it's fine that the underlying thing is a mutable map since upstream it isn't, but this just seems weird. We can set the map beforehand and then assign it on new `PartnerAd` or doing what vu suggests is also fine.